docs/source/getting_started.rst

```diff
@@ -36,20 +36,49 @@ in various programming languages. Smart contract APIs are available for Go, Node
 Hyperledger Fabric application SDKs
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Hyperledger Fabric offers a number of SDKs to support developing applications
-in various programming languages. SDKs are available for Node.js and Java:
+Hyperledger Fabric offers a number of SDKs to support developing applications in various programming languages. SDKs are available for Node.js and Java:
 
   * `Node.js SDK <https://github.com/hyperledger/fabric-sdk-node>`__ and `Node.js SDK documentation <https://hyperledger.github.io/fabric-sdk-node/>`__.
   * `Java SDK <https://github.com/hyperledger/fabric-gateway-java>`__ and `Java SDK documentation <https://hyperledger.github.io/fabric-gateway-java/>`__.
 
+  If you will be developing applications for Hyperledger Fabric leveraging the
+  Hyperledger Fabric SDK for Node.js, version 8 is supported from 8.9.4 and higher.
+  Node.js version 10 is supported from 10.15.3 and higher.
+
+    - `Node.js <https://nodejs.org/en/download/>`__ download
+
+  .. note:: Installing Node.js will also install NPM, however it is recommended
+            that you confirm the version of NPM installed. You can upgrade
+            the ``npm`` tool with the following command:
+
+  .. code:: bash
+  
+    npm install npm@5.6.0 -g
+
```

Comment:
I believe latest Node.js SDK uses Node 10 and 12, and npm 6.
In order to avoid these docs getting outdated again, I think we should simply point to the SDK docs and list the prereqs there.
The tutorials that utilize Node.js SDK should similarly point to SDK docs for the latest prereqs.

Reply:
I removed this content and simply link to the SDK READMEs for the latest pre-requisites.
I updated the Commercial paper and Writing your first app tutorials to link to the SDK READMEs as well. 
Also, per Per @negupta suggestion, I removed the Docker pre-req from the Commercial paper sample since it is already part of the Fabric pre-reqs here https://hyperledger-fabric.readthedocs.io/en/latest/prereqs.html#docker-and-docker-compose.